cev_eris.dme

```diff
@@ -2441,6 +2441,7 @@
 #include "code\modules\projectiles\guns\projectile\dartgun.dm"
 #include "code\modules\projectiles\guns\projectile\flaregun.dm"
 #include "code\modules\projectiles\guns\projectile\shotgun.dm"
+#include "code\modules\projectiles\guns\projectile\shotgun\ks23.dm"
```

Comment:
This should be in line 2501 not here, not sure if you mannully placed it or if DM messed it up

Reply:
manually placed, will change that now